pkgs/development/tools/godot/mono/make-deps.nix

```diff
@@ -0,0 +1,71 @@
+{ godot-mono
+, lib
+, nuget-to-nix
+, writeScriptBin
+, path
+, stdenv
+}:
+
+# This can be used to recreate deps.nix. This entails doing things that are not permitted
+# with a sandboxed Nix build, so instead of building the deps.nix directly, it builds a
+# script that can be run to generate it.  Thus, to use this derivation, simply do:
+#
+# $ nix run <path-to-nixpkgs>#godot-mono.make-deps [outdir]
+#
+# The work will all be done in the specified outdir, and the resulting deps.nix will be
+# placed in it at the end. If you don't provide an outdir, a temporary one will be created
+# in your tmpfs. The final line of output will specify the path to the resulting deps.nix.
+
+godot-mono.overrideAttrs (self: base: {
+  pname = "godot-mono-make-deps";
+
+  nativeBuildInputs = base.nativeBuildInputs ++ [ nuget-to-nix ];
+
+  outputs = [ "out" ];
+  phases = [ "installRunnerPhase" ];
```

Comment:
phases should never be overwriten like this.

Reply:
Why not?